pyaedt/edb_core/edb_data/padstacks_data.py

```diff
@@ -1018,12 +1018,39 @@ def _create_terminal(self, name=None):
     @pyaedt_function_handler
     def create_coax_port(self, name=None, radial_extent_factor=0):
         """Create a coax port."""
-        from pyaedt.edb_core.edb_data.ports import CoaxPort
 
-        term = self._create_terminal(name)
-        coax = CoaxPort(self._pedb, term._edb_object)
-        coax.radial_extent_factor = radial_extent_factor
-        return coax
+        port = self.create_port(name)
+        port.radial_extent_factor = radial_extent_factor
+        return port
+
+    @pyaedt_function_handler
+    def create_port(self, name=None, reference=None, is_circuit_port=False):
+        """Create a port on the padstack.
+
+        Parameters
+        ----------
+        name : str, optional
+            Name of the port.
```

Comment:
```suggestion
            Name of the port. The default is ``None``, in which case a name is automatically assigned.
```
The description of the behavior that occurs when ``None`` is used is a guess. Please revise if it is wrong.